discord/ui/action_row.py

```diff
@@ -143,7 +130,8 @@ def __init__(
     ) -> None:
         super().__init__()
         self._children: List[Item[V]] = self._init_children()
-        self._children.extend(children)
+        for child in children:
+            self.add_item(child)
         self._weight: int = sum(i.width for i in self._children)
```

Comment:
`add_item` fails because it needs `self._weight` to exist
```suggestion
        self._weight: int = sum(i.width for i in self._children)
        for child in children:
            self.add_item(child)
```